governator-legacy/src/main/java/com/netflix/governator/guice/InternalAutoBindModuleBootstrapModule.java

```diff
@@ -63,6 +63,7 @@ private Module bindAutoBindSingletons(BootstrapBinder binder) {
                         "@AutoBindSingleton(multiple=true) value cannot be set for Modules");
 
                 LOG.info("Found @AutoBindSingleton annotated module : {} ", clazz.getName());
+                LOG.info("***** @AutoBindSingleton for module {} is deprecated as of 10/10/2015.  Modules should be added directly to the injector or via install({}.class). See https://github.com/Netflix/governator/wiki/Auto-Binding", clazz.getName(), clazz.getSimpleName());
```

Comment:
I would tend to read that as saying something about that specific module is deprecated rather than AutoBindSingleton itself. Perhaps change the wording to something like:

> @AutoBindSingleton is deprecated as of Oct 10, 2015.  Module '{}' should be added directly to the injector or via install({}.class). See https://github.com/Netflix/governator/wiki/Auto-Binding

And as a minor change using a less [ambiguous date format](http://programmers.stackexchange.com/questions/42803/is-there-a-universal-date-format-that-anyone-in-the-world-can-understand).

For probable pit falls:

1. Since many internal users aren't using guice directly, it might be helpful to have the wiki cover the common ways to add modules for base-server, NFLifecycleUnitTester, etc. 
2. Is it assumed a given app owner will disable auto-binding as they switch? If so it would probably be good to add instructions to the wiki for how to do that. If not:
     * Do you think this should also check/warn about module deduping? Checking one of our modules that has AutoBindSingleton it doesn't currently implement equals/hashCode. If a user explicitly adds that module it will still get loaded via auto-binding for some time which will break with duplicate bindings.
     * Is there a way to detect and ignore auto-binding if the module is already bound? Otherwise the deprecation notice would keep showing up even though there is nothing to do.